src/server/page_server.cpp

```diff
@@ -368,6 +368,60 @@ page_server::connection_handler::push_disconnection_request ()
   push_request (page_to_tran_request::SEND_DISCONNECT_REQUEST_MSG, std::string ());
 }
 
+page_server::follower_connection_handler::follower_connection_handler (cubcomm::channel &&chn, page_server &ps)
+  : m_ps { ps }
+{
+  constexpr size_t RESPONSE_PARTITIONING_SIZE = 1; // Arbitrarily chosen
+
+  m_conn.reset (new follower_server_conn_t (std::move (chn),
+  {
+    {
+      follower_to_followee_request::SEND_DUMMY,
+      std::bind (&page_server::follower_connection_handler::receive_dummy_request, std::ref (*this), std::placeholders::_1)
+    },
+  },
+  followee_to_follower_request::RESPOND,
+  follower_to_followee_request::RESPOND,
+  RESPONSE_PARTITIONING_SIZE,
+  nullptr)); // TODO handle abnormal disconnection.
+
+  m_conn->start ();
+}
+
+void page_server::follower_connection_handler::receive_dummy_request (follower_server_conn_t::sequenced_payload
+    &&a_sp)
+{
+  er_log_debug (ARG_FILE_LINE, "A follower has requested a duumy request");
+}
+
+page_server::followee_connection_handler::followee_connection_handler (cubcomm::channel &&chn, page_server &ps)
+  : m_ps { ps }
+{
+  constexpr size_t RESPONSE_PARTITIONING_SIZE = 1; // Arbitrarily chosen
+
+  m_conn.reset (new followee_server_conn_t (std::move (chn),
+		{}, // followee doesn't request anything
+		follower_to_followee_request::RESPOND,
+		followee_to_follower_request::RESPOND,
+		RESPONSE_PARTITIONING_SIZE,
+		nullptr)); // TODO handle abnormal disconnection.
+
+  m_conn->start ();
+}
+
+void
+page_server::followee_connection_handler::push_request (follower_to_followee_request reqid, std::string &&msg)
+{
+  m_conn->push (reqid, std::move (msg));
+}
+
+int
+page_server::followee_connection_handler::send_receive (follower_to_followee_request reqid, std::string &&payload_in,
```

Comment:
nit;
maybe not justified to have this as a separate method; we'll extract one later if needed